src/lib/config.ts

```diff
@@ -17,6 +17,12 @@ export const CONFIG: Config = {
           description: 'Time to first response on issues',
           benchmarks: (weeks: number) => categorizeLowerIsBetter([1, 4, 12, 52], weeks),
         },
+        {
+          name: 'releaseFrequency',
+          weight: 3,
+          description: 'Number of releases in the past year',
+          benchmarks: (releases: number) => categorizeHigherIsBetter([55, 34, 5, 1], releases),
```

Comment:
for a different PR -> your README should include a table like this:

metric | 1 star | 2 stars | 3 stars | 4 stars | 5 stars 
---- | ---- | ---- | ---- | ---- | ----
release frequency | 0 | 1-4 | 5-33 | 34-54 | 55+